docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-enable-cluster-label-verification/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+description: "Migrating Mimir Helm Installation to Use Memberlist Cluster Label"
+title: "Migrating Mimir Helm Installation to Use Memberlist Cluster Label"
+menuTitle: "Migrating Mimir Helm Installation to Use Memberlist Cluster Label"
+weight: 110
+---
+
+# Migrating Mimir Helm Installation to Use Memberlist Cluster Label
+
+## Introduction
+
+Grafana Mimir uses [memberlist] to share works and deciding which component replica to send the workload such as when ingesting series. Memberlist encodes the replica information in data structure called [hash ring], this is a consistent hashing scheme in which different ingesters instance tokens are placed around the ring. The token position in the ring determines which ingester should handle a request. The information of this hash ring is delivered to different components by using gossip protocol.
+
+By default, hash ring is global. If we also have Grafana Tempo and Grafana Loki in the same Kubernetes cluster, their components can unintentionally talk and sending data with each other, because Loki and Tempo also uses memberlist. This possibility to happen increases in cluster where the pods may reuse each other's IPs after churning.
+
+In this document we will describe the step on how to migrate a Mimir installation to prevent two separate memberlist gossip hash ring to join into one.
+
+There are three steps of the migration which we will describe in details in the migration section. But in brief summary the steps are:
+
+1. Disable memberlist cluster label verification
+1. Set cluster label to all Mimir components
+1. Enable memberlist cluster label verification again
+
+The migration should take around 30 minutes. The risk of not doing this migration if you have Mimir and Tempo or Loki in the same cluster is the possibility of those different backend to talk with each other.
```

Comment:
I think the second sentence on L24 is the most important bit of this introduction. A reader wants to know why they should perform this task and they may not really care about the conceptual information on L12.

Reply:
I removed many repetition and make a more concise introduction in the beginning of the section.